### ISSUE TITLE
[vite-plugin] Vite 3 auto import components and <script setup>

### ISSUE THREAD

**Issue by meixger** [vuetifyjs/vuetify-loader#260]
I cannot get the **production** bundle to **auto import the components** when using **`<script setup>`** syntax.

![vappbar-broken](https://user-images.githubusercontent.com/600424/180266203-a0ca199a-ea64-44df-8a4f-162586dd9f20.png)

It works fine in production without `<script setup>` syntax. In dev mode it's working with either syntax.

Is this not (yet) supported?

Here is a [simple demo repo](https://github.com/meixger/Vuetify3ScriptSetup) with screenshots.

Versions:
- Vue 3.2.37
- Vuetify 3.0.0-beta.5
- Vite 3.0.2
- vite-plugin-vuetify 1.0.0-alpha.12

**Comment by KaelWD** [comment]
Duplicate of #256

**Comment by meixger** [comment]
Resolved only after [1.0.0-alpha.14](https://github.com/vuetifyjs/vuetify-loader/releases/tag/vite-plugin-vuetify%401.0.0-alpha.14) in https://github.com/vuetifyjs/vuetify-loader/commit/322f6ba511c0da4ffbb90e49bd7d467d993b8ac6
